src/inspector_agent.h

```diff
@@ -21,6 +19,26 @@ class Message;
 namespace node {
 namespace inspector {
 
+class InspectorOptions {
+ public:
+  InspectorOptions(): wait_connect_(false), http_enabled_(false),
+                      host_("127.0.0.1"), port_(9229) { }
+#if HAVE_INSPECTOR
+  bool ParseOption(const char* option);
+#endif
+  bool InspectorEnabled() const { return HttpEnabled(); }
+  bool HttpEnabled() const { return http_enabled_; }
+  bool WaitForConnect() const { return wait_connect_; }
+  std::string HttpHost() const { return host_; }
```

Comment:
I realize this follows existing convention but HttpHost() is something of a misnomer since it's an address, not a free-form host name.

Aside: simple getters are usually called after the property they encapsulate minus the trailing underscore.